src/Forside/SammenligningMedBransje/ForklaringAvPeriode.tsx

```diff
@@ -1,20 +1,67 @@
 import React, {FunctionComponent, ReactElement} from 'react';
 import {Ingress, Normaltekst} from 'nav-frontend-typografi';
-import {getAriaLabelTekstForOppdateringsdato, periodeFraOgTil} from '../../utils/app-utils';
+import {
+  formatterDatoMedMånedNavn,
+  getPeriodeMedDato,
+} from '../../utils/app-utils';
 import './ForklaringAvPeriode.less';
 import {SammenligningsType} from '../vurderingstekster';
 import classNames from 'classnames';
+import {
+  RestPubliseringsdatoer
+} from "../../api/publiseringsdatoer-api";
+import {RestStatus} from "../../api/api-utils";
+import NavFrontendSpinner from "nav-frontend-spinner";
 
 interface Props {
   sammenligningsType: SammenligningsType;
   className?: string;
+  restPubliseringsdatoer: RestPubliseringsdatoer;
 }
 
 export const ForklaringAvPeriode: FunctionComponent<Props> = (
     {
       sammenligningsType,
       className,
+      restPubliseringsdatoer,
     }) => {
+
+  const getPubliseringsdatoinfoElement = (): ReactElement => {
+    if (restPubliseringsdatoer.status === RestStatus.Suksess) {
+      const publiseringsdatoer = restPubliseringsdatoer.data;
+      return (
+          <div>
+            <Normaltekst>{`Sist oppdatert: `+ formatterDatoMedMånedNavn(new Date(publiseringsdatoer.sistePubliseringsdato))}</Normaltekst>
+            <Normaltekst>{`Neste oppdatering: `+ formatterDatoMedMånedNavn(new Date(publiseringsdatoer.nestePubliseringsdato))}</Normaltekst>
+          </div>
+      )
+    } else if (
+        restPubliseringsdatoer.status === RestStatus.LasterInn ||
+        restPubliseringsdatoer.status === RestStatus.IkkeLastet
+    ) {
+      return <NavFrontendSpinner/>
+    } else {
+      return <Normaltekst>{""}</Normaltekst>
+    }
+  }
+
+  const getPeriodeElement = (): ReactElement => {
```

Comment:
Denne funkjonen er en React-komponent, så i følge navenstandarden for kompoenter bør den hete `Periode` el